src/common/constants/organization.js

```diff
@@ -1,6 +1,28 @@
-// eslint-disable-next-line import/prefer-default-export
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+
 export const ORGANIZATION_STATUS = {
   active: 'Active',
   inactive: 'Inactive',
   pending: 'Pending',
 };
+
+export const ORGANIZATION_STATUS_LABELS = {
+  [ORGANIZATION_STATUS.active]: <FormattedMessage id="ui-organizations.organizationStatus.active" />,
+  [ORGANIZATION_STATUS.inactive]: <FormattedMessage id="ui-organizations.organizationStatus.inactive" />,
+  [ORGANIZATION_STATUS.pending]: <FormattedMessage id="ui-organizations.organizationStatus.pending" />,
+};
+
+export const ORGANIZATION_PAYMENT_METHOD = {
```

Comment:
it looks like we have the same constants in ui-invoice. maybe it makes sense to leverage them into stripes-acq-components https://github.com/folio-org/ui-invoice/blob/d39fe77dd3511486f8c3541a0ac0b9eeddd54fba/src/common/constants/constants.js#L27

Reply:
invoices have more options